cli_reference/cli_manager/cli-manager-install.adoc

```diff
@@ -0,0 +1,22 @@
+:_content-type: ASSEMBLY
+[id="cli-manager-install_{context}"]
+= Installing the {cli-manager}
+include::_attributes/common-attributes.adoc[]
+:context: cli-manager-install
+
+toc::[]
+
+//Brief explanation of what the cli manager is.
+The {cli-manager} can be used in both connected and disconnected environments, and particularly makes installing and managing CLI plugins in disconnected environments easier. It is makes Krew compatible with the `oc` CLI and cluster administrators add CLI plugin custom resources which can then be accessed in both connected and disconnected environments.
```

Comment:
Typo, but also a question - I thought Krew was already compatible with the oc CLI

```suggestion
The {cli-manager} can be used in both connected and disconnected environments, and particularly makes installing and managing CLI plugins in disconnected environments easier. It makes Krew compatible with the `oc` CLI and cluster administrators add CLI plugin custom resources which can then be accessed in both connected and disconnected environments.
```

Reply:
@ardaguclu What do you think?

Reply:
yes, Krew is already compatible with oc. What CLI Manager does additionally is to serve a Krew compatible custom index. 
You are the experts but this statement would be closer to be correct;

```
It serves a Krew compatible custom index and cluster administrators add CLI plugin custom resources which can then be accessed in both connected and disconnected environments.
```